components/trade/TradeActivityTable.vue

```diff
@@ -7,7 +7,7 @@
           :key="filter.id"
           :active="filter.id === activeTab"
           variant="outlined-rounded"
-          data-testid="profile-activity-button-filter"
+          :data-testid="`trade-tab-${filter.id}`"
```

Comment:
It's no need to set `data-testid` here because there are no related tests cases.

Reply:
done